### PR TITLE
issue123 changes from base repo

### PR DESCRIPTION
**Pull request by Malarg**

### CODE REVIEW

**Comment by nausharipov** [comment]
Is there an issue this PR needs to be linked to?

File: lib/src/code_modifiers/indent_code_modifier.dart [hunk 1]
```diff
@@ -5,10 +5,10 @@ import 'package:flutter/widgets.dart';
 import '../code_field/editor_params.dart';
 import 'code_modifier.dart';
 
-class IntendModifier extends CodeModifier {
+class IntentModifier extends CodeModifier {
   final bool handleBrackets;
 
-  const IntendModifier({
+  const IntentModifier({
```
Comment (nausharipov) [inline lib/src/code_modifiers/indent_code_modifier.dart:11]: The file name is `indent_code_modifier`, while the class is renamed to `IntentModifier`.

File: lib/src/code_field/code_controller.dart [hunk 2]
```diff
@@ -131,14 +102,11 @@ class CodeController extends TextEditingController {
     this.stringMap,
     this.params = const EditorParams(),
     this.modifiers = const [
-      IntendModifier(),
+      IntentModifier(),
```
Comment (alexeyinkin) [inline lib/src/code_field/code_controller.dart:105]: `IndentModifier`

File: lib/src/code_field/code_controller.dart [hunk 3]
```diff
@@ -361,10 +311,6 @@ class CodeController extends TextEditingController {
     //   newValue = newValue.copyWith(text: _spacesToMiddleDots(newValue.text));
     // }
```
Comment (alexeyinkin) [inline lib/src/code_field/code_controller.dart:312]: Delete this too.

**Comment by alexeyinkin** [comment]
Did you run the full comparison like this?

image

It can reveal minor differences that can be ironed out like this blank line. Also this would prevent the `IntentModifier` typo.

Please rerun this way.

**Comment by alexeyinkin** [comment]
image

**Review by alexeyinkin — CHANGES_REQUESTED** [review]
I ran the full diff and saw no more to borrow except on the screens above. So you do not need to full-diff again.

File: lib/src/code_modifiers/indent_code_modifier.dart (hunk 1 above)
Comment (Malarg) [reply lib/src/code_modifiers/indent_code_modifier.dart:11]: file named such way in a base repo. @alexeyinkin What do you think? Are we still support compatibility with the base repo?

File: lib/src/code_field/code_controller.dart (hunk 2 above)
Comment (Malarg) [reply lib/src/code_field/code_controller.dart]: fixed

File: lib/src/code_field/code_controller.dart (hunk 3 above)
Comment (Malarg) [reply lib/src/code_field/code_controller.dart]: done

**Review by nausharipov — APPROVED** [review]

**Review by alexeyinkin — APPROVED** [review]

**Pull request merged**